### PR TITLE
ubuntu 18.10 build instructions

### PR DESCRIPTION
**Pull request by codedumper1**
steps to build rippled.

### CODE REVIEW

**Comment by codedumper1** [comment]
I signed it!

File: README.md [hunk 1]
```diff
@@ -13,3 +13,32 @@ Ripple supports replication to/from MariaDB and MySQL using GTIDs (of MariaDB
...
+### Build
+```
+cd mysql-ripple
+bazel build :all
+bazel test
+
+```
+If all went OK, your binary is at
+```
+find $HOME -type f -name rippled
```
Comment (pivanof) [inline README.md]: This is inconsistent with the rest of the instructions (there's no mention of $HOME anywhere else). I think this should say "your binary is at ./bazel-bin/rippled" instead.

File: README.md [hunk 2]
```diff
@@ -13,3 +13,32 @@ Ripple supports replication to/from MariaDB and MySQL using GTIDs (of MariaDB
...
+### Clone source
+```
+git clone https://github.com/google/mysql-ripple.git
+```
+
+### Build
+```
+cd mysql-ripple
+bazel build :all
+bazel test
```
Comment (dveeden) [inline README.md]: Shouldn't that be `bazel test :all` ?

File: README.md (hunk 1 above)
Comment (codedumper1) [reply README.md]: I fixed it according to your recommendation.

File: README.md (hunk 2 above)
Comment (pivanof) [reply README.md]: @codedumper1 can you fix this? Then I'll be able to merge the PR.

Comment (codedumper1) [reply README.md]: @pivanof I changed test line as you requested.

**Pull request merged**